Quick Start Documentation/Instrument Addon Quick Start Guide.md

```diff
@@ -0,0 +1,290 @@
+
+Use Message Status and Error Code to sequence through a series of Messages.<br/><br/>
+For example, trigger a Message and wait for Message Status to be true. This means that the Message was executed.<br/>
+Then check Error Code. If Error Code is 0, the Message was successfully sent.<br/>
+Then go on to trigger the next Message.​ Timestamp can be used in a similar fashion.<br/>
+Trigger a Message, wait for Timestamp to change, check Error Code.
+
+
+## Add-on Tool
+
+​The add-on also comes with a host tool that provides additional features. These features help in manual testing and debugging. Add this tool to the project files:
+
+![Instrument Tool](./Images/image029.png)
+
+It can also be added as a button to the UI screen.
+
+![Instrument Tool UI Screen](./Images/image031.png)
+
+Click open on the file or select the configured button to launch the tool.​
+
+![Instrument Tool Launch](./Images/image033.png)
+
+Select File >> Configure and select the desired add-on.​
+
+![Instrument Tool Configure](./Images/image034.png)
+
+​Then, put in any string Command to send, configure the Response criteria, and send the raw message.<br/>
+The Response string will be returned in the Response field.<br/><br/>
+Note: this goes around all the VeriStand channels. They will not update as part of this.<br/>
+This provides a place to test and debug the instrument for things like basic connection, termination characters, message format​, etc.
+
+![Instrument Tool Command](./Images/image035.png)
+
+## Host Automation API
+
+In addition to the above items, there is an automation API that can be used from Windows to initiate commands with the add-on. This is installed at:<br/>
+C:\Users\Public\Documents\National Instruments\NI VeriStand <Year>\Custom Devices\Instrument Addon\Windows\Host Automation API<br/><br/>
+An example of using this API is as follows:
+
+![Host API Example](./Images/image036.png)
+
+First, connect to the add-on by its name and expected target IP address. Then send raw messages. When finished, close the connection.
+
+## System Definition API
+
+There is also a system definition API that allows the user to programatically create an instance of the add-on from a previously exported configuration.<br/>
+This is installed at:<br/>
+C:\Users\Public\Documents\National Instruments\NI VeriStand <Year>\Custom Devices\Instrument Addon\Windows\System Definition API<br/><br/>
+An example of using this API is as follows:​​​
+
+![System Definition API Example](./Images/image038.jpg)
+
+## Scripting API
+
+Users can add flexibility and reusability to their systems by creating and configuring system definitions through the Scripting API.
+
+## With this API, users can:<br/>
+- create a new Instrument from a previously exported configuration:
+
+    Node used: **Create Instrument Addon From Configuration File**
+
+![Create Instrument Addon From Configuration File](./Images/image040.png)
+
+- create a new Instrument from scratch<br/>
```

Comment:
A new line needs to be added here for better rendering and consistency with the other examples.

Reply:
I'm ok with the changes, but just a new line here would make a huge visual difference.

![image](https://user-images.githubusercontent.com/62251693/94819314-c7e6b580-0407-11eb-870f-83256dd93bbd.png)